draft-ietf-cats-framework.md

```diff
@@ -159,7 +153,15 @@ Service:
   : The same service can be provided in many locations; each of them constitutes a service instance.
 
 Computing Service:
-  : An offering is made available by a provider by orchestrating a set of computing resources (without networking resources).
+  : An offering that is made available by a provider by orchestrating a set of computing resources to process application content.
```

Comment:
Do we really need to mention "to process application content"?

Reply:
this is for sync up with the definition in use case draft. I am good with removing or keeping it. But if we want to address the comment, better to keep it. LOL. Only some editorial modification.